.gitignore

```diff
@@ -1,3 +1,7 @@
+# derp
+media
+migrations/
```

Comment:
Don't omit the migrations files from your repository.  Those files are part of the history of your data model development process and are important code.